manifests/init.pp

```diff
@@ -82,6 +82,10 @@
   $renew_cron_hour                       = $letsencrypt::params::renew_cron_hour,
   $renew_cron_minute                     = $letsencrypt::params::renew_cron_minute,
   $renew_cron_monthday                   = $letsencrypt::params::renew_cron_monthday,
+  $systemd_date_spec                     = $letsencrypt::params::systemd_date_spec,
+  Enum['cron', 'systemd'] $timer_type    = $letsencrypt::params::timer_type,
+  Boolean          $manage_firewall      = $letsencrypt::params::manage_firewall,
+  Optional[Array[String[1]]]  $manage_services      = $letsencrypt::params::manage_services,
```

Comment:
Can you explain what this parameter is expected to do?

Reply:
We already manage in post and pre hook . Not sure it is safe to have to maintain two differents methods in a same module to make the same task.

Reply:
The current method can handle a list of command for hooks. And i don't think using two different methods for a same mechanism is a good thing.